frontend/containers/profile-header/component.tsx

```diff
@@ -51,12 +51,12 @@ export const ProfileHeader: FC<ProfileHeaderProps> = ({
 
   return (
     <div className={className}>
-      <div className="py-10 mx-4 bg-center bg-cover lg:mx-0 lg:px-4 lg:py-18 rounded-2xl bg-radial-green-dark bg-green-dark">
+      <div className="py-10 bg-center bg-cover md:mx-4 md:px-4 md:py-18 md:rounded-2xl bg-radial-green-dark bg-green-dark min-h-[425px] md:min-h-fit">
```

Comment:
I think we should add back `lg:mx-0`, don't you think? That way we'll align the profile header with the rest of the page (as it is right now on staging).

<img src="https://user-images.githubusercontent.com/6073968/196113352-c18b6dc6-aec2-4bb9-9931-fc071d1589a3.png" alt="The profile header's horizontal padding is highlighted in red" />